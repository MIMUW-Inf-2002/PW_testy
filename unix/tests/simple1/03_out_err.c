#include <unistd.h>
#include <stdio.h>

#define N 10

const int period = 100000; // 100 milliseconds

int main() {
    fprintf(stderr, "err line %d\n", 1);
    fflush(stderr);
    fprintf(stdout, "out line %d\n", 1);
    fflush(stdout);

    fprintf(stderr, "err line %d\n", 2);
    fflush(stderr);
    fprintf(stdout, "out line %d\n", 2);
    fflush(stdout);

    return 0;
}