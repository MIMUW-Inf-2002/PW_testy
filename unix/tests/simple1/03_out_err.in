run ./03_out_err
run ./03_out_err
run ./03_out_err
run ./03_out_err
run ./03_out_err
run ./03_out_err
run ./03_out_err
run ./03_out_err
run ./03_out_err
run ./03_out_err
sleep 500
out 0
out 1
out 2
out 3
out 4
out 5
out 6
out 7
out 8
out 9
err 0
err 1
err 2
err 3
err 4
err 5
err 6
err 7
err 8
err 9
