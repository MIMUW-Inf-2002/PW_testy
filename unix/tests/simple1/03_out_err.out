======
Task 0 started: pid 53355.
Task 1 started: pid 53357.
Task 2 started: pid 53359.
Task 3 started: pid 53367.
Task 4 started: pid 53369.
Task 0 ended: status 0.
Task 3 ended: status 0.
Task 5 started: pid 53382.
Task 6 started: pid 53384.
Task 7 started: pid 53386.
Task 2 ended: status 0.
Task 8 started: pid 53388.
Task 4 ended: status 0.
Task 1 ended: status 0.
Task 9 started: pid 53398.
Task 8 ended: status 0.
Task 5 ended: status 0.
Task 6 ended: status 0.
Task 7 ended: status 0.
Task 9 ended: status 0.
======
Task 0 stdout: 'out line 2'.
Task 1 stdout: 'out line 2'.
Task 2 stdout: 'out line 2'.
Task 3 stdout: 'out line 2'.
Task 4 stdout: 'out line 2'.
Task 5 stdout: 'out line 2'.
Task 6 stdout: 'out line 2'.
Task 7 stdout: 'out line 2'.
Task 8 stdout: 'out line 2'.
Task 9 stdout: 'out line 2'.
Task 0 stderr: 'err line 2'.
Task 1 stderr: 'err line 2'.
Task 2 stderr: 'err line 2'.
Task 3 stderr: 'err line 2'.
Task 4 stderr: 'err line 2'.
Task 5 stderr: 'err line 2'.
Task 6 stderr: 'err line 2'.
Task 7 stderr: 'err line 2'.
Task 8 stderr: 'err line 2'.
Task 9 stderr: 'err line 2'.
